tests/shh-module/test_shh_filter.py

```diff
@@ -0,0 +1,21 @@
+import random,string,gevent
+
+def test_shh_filter(web3,skip_if_testrpc):
+    skip_if_testrpc(web3)
+    topic = "".join(random.choice(string.ascii_uppercase + string.digits + string.ascii_lowercase) for _ in range(10))
```

Comment:
Instead of using `random` here can we use something deterministic or non-random.  I have sound that unless the test is intentionally [fuzzing](https://en.wikipedia.org/wiki/Fuzz_testing) the inputs that randomness can lead to infuriating hard to reproduce test failures.